libs/components/src/popover/vl-popover-action.component.ts

```diff
@@ -0,0 +1,37 @@
+import { CSSResult, PropertyDeclarations, TemplateResult, html, nothing } from 'lit';
+import { BaseLitElement } from '@domg-wc/common-utilities';
+import { resetStyle } from '@domg/govflanders-style/common';
+import { vlElementsStyle } from '@domg-wc/elements';
+import { customElement } from 'lit/decorators.js';
+import popoverActionUigStyle from './vl-popover-action.uig-css';
+
+@customElement('vl-popover-action')
+export class VlPopoverActionComponent extends BaseLitElement {
+    public icon = '';
```

Comment:
ook op andere plaatsen: de conventie is public niet te schrijven
probeer private te maken wat kan